deb-packaging/src/deb/control/postrm

```diff
@@ -15,4 +15,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-/usr/sbin/userdel brooklyn
+if /usr/bin/getent passwd brooklyn; then
+    /usr/sbin/userdel brooklyn
```

Comment:
IMO its better to leave the user alone ... what if it had been created before install?